packages/react-ui-kit/src/Form/Select.tsx

```diff
@@ -25,19 +25,20 @@ import {filterProps} from '../util';
 import {inputStyle} from './Input';
```

Comment:
this file is so big, can we figure out a way to split it up.

Reply:
im sort of thinking about an abstracted general select and then two wrappers, one for single input select and one for multi-select. it could make the whole family more reusable and more readable